commands/operator-sdk/cmd/build.go

```diff
@@ -163,25 +164,27 @@ func buildFunc(cmd *cobra.Command, args []string) {
 		baseImageName += "-intermediate"
 	}
 	dbcmd := exec.Command("docker", "build", ".", "-f", "build/Dockerfile", "-t", baseImageName)
-	o, err := dbcmd.CombinedOutput()
+	dbcmd.Stdout = os.Stdout
+	dbcmd.Stderr = os.Stderr
+	err = dbcmd.Run()
 	if err != nil {
 		if enableTests {
-			log.Fatalf("failed to build intermediate image for %s image: %v (%s)", image, err, string(o))
+			log.Fatalf("failed to build intermediate image for %s image: %v", image, err)
 		} else {
-			log.Fatalf("failed to output build image %s: %v (%s)", image, err, string(o))
+			log.Fatalf("failed to output build image %s: %v", image, err)
 		}
 	}
-	fmt.Fprintln(os.Stdout, string(o))
 
 	if enableTests {
 		testBinary := filepath.Join(wd, scaffold.BuildBinDir, filepath.Base(wd)+"-test")
 		buildTestCmd := exec.Command("go", "test", "-c", "-o", testBinary, testLocationBuild+"/...")
 		buildTestCmd.Env = goBuildEnv
-		o, err := buildTestCmd.CombinedOutput()
+		buildTestCmd.Stdout = os.Stdout
```

Comment:
If we are going to print building the intermediate image, I don't see us log to the user that is what is happening. I think it might be nice to do that so when they see two docker builds happening they are not confused. Thoughts?

Reply:
We don't log anything right now unless there's a failure.  If we add a log for one of the steps, do you think it makes sense to go ahead and log all of the steps.

Reply:
^^ Thoughts?

We're currently printing the intermediate image build output without a log saying what's happening, so this PR doesn't change that.

I agree that adding some logging for context would be helpful though. Would logging before every subprocess be unnecessarily verbose? 

Without `--enable-tests`, the only output is the docker build output for the final image, so users should understand that with no additional context. With `--enable-tests`, the output is the docker build output for both the intermediate and final image. So maybe we only log both docker builds, but only when tests are enabled?

Reply:
Sounds good to me. In that case, I'll merge this as is.